main.js

```diff
@@ -68,6 +70,26 @@ $(function() {
       $(codeLines[start - 1]).find('.ellipsis').remove();
     }
   }
+  
+  const foldAll = () => {
+    $('.collapser:not(:first)').addClass('sideways');
+    
+    pairs.forEach((end, start) => {
+      if (start !== 0) {
```

Comment:
The `start` here refers to the line number, not the index of an array. So if your intent is to ignore the first block, it won't accomplish that unless the first line of this block is 0.